Documentatie/EnergielabelApi-validaties.md

```diff
@@ -103,28 +104,27 @@ Onderstaande validaties worden allemaal en in willekeurige volgorde uitgevoerd.
+|:------------------|:------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+|	Toevoegen	|	<ul><li>Vergunningsaanvraag</li><li>Oplevering</li><li>Bestaand</li></ul>	|<ul><li> Op de opnamedatum (SurveyDate) dient voor geen enkele van de opnameadressen al een registratie te zijn met dezelfde scope en gebouwklasse.</li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ kan enkele één van de blokken ‘BAGIdentification’ of ‘ProvisionalIdentification’ gekozen worden. </li><li> Toevoegen voor de status ‘Vergunningsaanvraag’ is het niet mogelijk om te registreren met een ‘provisionalid’. </li><li> Toevoegen voor woningbouw met een detailaanduiding mag alleen met rechten. Deze kunt u aanvragen in EP-Online. </li></ul>
+|	Vervangen		|	<ul><li>Vergunningsaanvraag</li><li>Oplevering</li><li>Bestaand</li></ul>	|	<ul><li>Op de opnamedatum (SurveyDate) dient er al een registratie te zijn met dezelfde scope en gebouwklasse, waarbij alle adressen overeen komen met de initiële registratie.</li><li>Vervangen mag alleen wanneer de rekenmethodiek versie overeen komt met de versie die gebruikt is in de initiële registratie.</li><li>Vervangen mag niet wanneer een of meer van de gebouwen binnen de registratie ingetrokken is.</li><li>Vervangen kan alleen gedaan worden door de registratieadviseur die de initiële registratie heeft gedaan.</li><li>Vervangen van status ‘Vergunningsaanvraag’ kan zonder rechten gedaan worden binnen 60 dagen vanaf de originele registratiedatum. Hierna zijn rechten nodig. Deze kunt u aanvragen in EP-Online.</li><li>Vervangen van status ‘Oplevering’ of status ‘Bestaand’ kan zonder rechten gedaan worden binnen 7 dagen vanaf de originele registratiedatum. Hierna zijn rechten nodig. Deze kunt u aanvragen in EP-Online.</li></ul>
+|	Uitbreiden		|	<ul><li>Oplevering</li><li>Bestaand</li></ul>		|	<ul><li> Op de opnamedatum (SurveyDate) dient er al een registratie te zijn met dezelfde scope en gebouwklasse, waarbij alle adressen overeen komen met de initiële registratie.</li><li> Uitbreiden mag alleen wanneer de meeste recente rekenmethodiek versie wordt gebruikt.</li><li> Uitbreiden mag alleen wanneer de rekenmethodiek versie overeen komt met de versie die gebruikt is in de initiële registratie.</li><li> Uitbreiden mag alleen voor status ‘Oplevering’ en status ‘Bestaand’. </li><li> Uitbreiden mag aleen wanneer de adressen waarmee uitgebreid wordt/worden nog geen recentere registratie hebben. </li><li> Uitbreiden voor woningbouw met een detailaanduiding mag alleen met rechten. Deze kunt u aanvragen in EP-Online. </li></ul> 
```

Comment:
"Uitbreiden mag alleen..." Bij toevoegen, herlabelen en vervangen ook dezelfde opbouw van de zin gebruiken voor structuur, consistentie en leesbaarheid.

Mijn voorkeur ligt bij ".... is alleen mogelijk..."